<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Magazyn&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_kontrahenci">
     <attribute name="title">
      <string>Kontrahenci</string>
     </attribute>
     <widget class="QTableView" name="tableView_kontrahenci">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>990</width>
        <height>400</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_kontrahenci">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>990</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Wyszukaj</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_dodajKontrahenta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj Kontrahenta</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nazwaKontrahenta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Nazwa Firmy / Imie Nazwisko</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nip">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>NIP / PESEL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_tel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Numer Telefonu</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ulica">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>570</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ulica i numer budynku/mieszkania</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_kod_pocztowy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Kod pocztowy</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Miasto">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Miasto</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_dostawca">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>660</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dostawca</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_odbiorca">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>660</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Odbiorca</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_dodajKontrahenta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>690</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj Kontrahenta</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_komunikatDodaniaKontrahenta">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>720</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_usunKontrahenta">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>450</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuń Kontrahenta</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nazwaKontrahenta_usun">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>480</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Nazwa Firmy / Imie Nazwisko</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nipkontrahenta_usun">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>510</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>NIP / PESEL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_usunKontrahenta">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>540</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuń Kontrahenta</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Magazyn</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
